README.md

```diff
@@ -43,6 +43,7 @@ Streamline the deployment of your serverless applications to Azure Functions, an
 For containerized apps (single- or multi-containers) to create a complete workflow 
 - use [Docker login](https://github.com/Azure/docker-login)(`azure/docker-login`) 
 to build container images, push to a container registry (Docker Hub or Azure Container Registry) and then deploy the images to a Azure Web App or Azure Function for Containers, or to Kubernetes. 
+- use [Deploy to Azure Container Instances](https://github.com/Azure/aci-deploy-action)(`azure/aci-deploy-action`) to deploy your container images to Azure Container Instances.

```

Comment:
You could rename your action's repo to `azure/aci-deploy` instead maybe? Given that none of the other actions use the `-action` suffix.

Reply:
@geverghe To take the final call on this

Reply:
Renamed the Action's repo to aci-deploy. Will update my changes too